src/devtools/useAtomDevtools.ts

```diff
@@ -1,7 +1,6 @@
 import { useEffect, useRef } from 'react'
 import { useAtom } from 'jotai'
-import type { WritableAtom } from 'jotai'
-import type { Scope } from '../core/atom'
+import type { Atom, Scope, SetAtom, WritableAtom } from '../core/atom'
```

Comment:
```suggestion
import type { Atom, WritableAtom } from 'jotai'
import type { Scope, SetAtom } from '../core/atom'
```
Let's make it sure what's public and what's internal.